### ISSUE TITLE
A11y - Terminal theme ANSI theme colors are inconsistent in light themes 

### ISSUE THREAD

**Issue by pwang347** [microsoft/vscode#204883]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.86.1
- OS Version: Mac Sonoma 14.3

Steps to Reproduce:

Note: the following is just one way to easily reproduce, but this really is an issue with the CSS variable that affects more than just the Jupyter extension (hence why the issue is opened in this repository).

1. Install the Jupyter extension
2. In a cell, execute code `asd`
3. Note that in the cell output, the green color is using the theme variable `--vscode-terminal-ansiGreen`
<img width="263" alt="image" src="https://github.com/microsoft/vscode/assets/8560030/468ca443-4ce5-42bf-8b2c-952125b614e5">

4. Run the following in the terminal `echo -e "\033[0;32mGREEN\033[0m"`
5. Observe that the green ANSI styling is different in light themes:
<img width="280" alt="image" src="https://github.com/microsoft/vscode/assets/8560030/00f23a87-20dc-48e9-95e9-248a2535226c">
<img width="427" alt="image" src="https://github.com/microsoft/vscode/assets/8560030/f0a9bb62-981c-4343-bb0f-29033e9ccd42">
<img width="419" alt="image" src="https://github.com/microsoft/vscode/assets/8560030/db388e86-f78b-471f-bc31-786b549e1664">

Works as expected in dark themes:
<img width="377" alt="image" src="https://github.com/microsoft/vscode/assets/8560030/e8ceffd6-bbd3-4e4d-8c04-44a14fda625f">
<img width="290" alt="image" src="https://github.com/microsoft/vscode/assets/8560030/111e167b-759d-4371-845d-2ffc7ef8b02c">

This is a problem because in light theme, the contrast ratio requirement is not met. This means that when the CSS variable is used there will be an accessibility bug.

Thanks!

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.86.1. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by pwang347** [comment]
Confirmed reproing on 1.86.1 as well

**Comment by Yash14j** [comment]
**Github Tags:**

#BM-VisualStudioCodeClient-Win32-Jan2024; #Visual Studio Code Client; #A11yMAS; #WCAG1.4.3; #AILimited; #A11yTCS; #ColorContrast;

**Comment by Tyriar** [comment]
Assigned to April as it comes from s360

**Comment by daviddossett** [comment]
@Tyriar so no matter what color we use, terminal will automatically adjust it? I can change the default color used to solve the contrast issue outside of terminal. Just not sure how to keep them aligned.

**Comment by Tyriar** [comment]
@daviddossett yes the terminal will by default slide the color to be 4.5:1 (`terminal.integrated.minimumContrastRatio`). It's fine if their luminosity is a little inconsistent imo, we just need to make sure the color meets contrast elsewhere.

**Comment by daviddossett** [comment]
Tweaked colors for light and light hc themes in #209997

![CleanShot 2024-04-09 at 16 55 31@2x](https://github.com/microsoft/vscode/assets/25163139/ad39ca31-1de4-4d46-91f2-b345e91f2667)

**Comment by isidorn** [comment]
@daviddossett thanks a lot. I think we can close this bug as fixed now, right?

**Comment by daviddossett** [comment]
Build failing, will auto close when it merges
